settings.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+GITHUB_WEBHOOK_SECRET = os.environ['GITHUB_WEBHOOK_SECRET']
+GITHUB_ACCESS_TOKEN = os.environ['GITHUB_ACCESS_TOKEN']
+
+WATCH_CONFIG = {
+    'edx/edx-platform': {
+        'patterns': ['doc/*', ],
+        'recipients': ['nobody@example.com', ],
+        'subject': '[ADR] Pull request: {repo} #{number} {action}',
+    },
+}
```

Comment:
@pomegranited, I was planning to do that, but since I was wary of the timebox at that time, I decided against doing it. But since I have time and this shouldn't take a lot of time, I will fix this.

Reply:
Thank you!  Works great.